packages/block-editor/src/components/block-rename/modal.js

```diff
@@ -81,7 +107,7 @@ export default function BlockRenameModal( {
 					<TextControl
 						__nextHasNoMarginBottom
 						__next40pxDefaultSize
-						value={ editedBlockName }
+						value={ editedBlockName || blockName }
```

Comment:
Because there is a fallback block name, it seems that it is not possible to make the block name empty, i.e. revert to the default block name:


https://github.com/user-attachments/assets/10706818-4330-496d-8153-a053681c17a2

Probably it has to do with refactoring state logic, simply deleting || `blockName` doesn't seem to work. Any ideas?

Reply:
We're testing whether it's submittable, but we may also need to test whether the text field can be left empty:

https://github.com/WordPress/gutenberg/blob/7f82629d9a52ebc151f5d56122be0b54fe22654a/test/e2e/specs/editor/various/block-renaming.spec.js#L124-L133

Reply:
Good catch, @t-hamano! I'll refactor the logic and update or add a new e2e test.

Reply:
Should be fixed via [11a3660](https://github.com/WordPress/gutenberg/pull/68560/commits/11a3660d8da8a8f5c4fbcd6e06a7a693e4df53c1).